packages/gatsby-plugin-typescript/src/gatsby-node.js

```diff
@@ -33,6 +39,38 @@ function onCreateWebpackConfig({ actions, loaders }) {
       ],
     },
   })
+
+  if (stage === `develop`) {
+    let ts
```

Comment:
Please use a more descriptive name
```suggestion
    let isTypescriptDepAvailable
```